examples/app-basic/src/components/App/App.tsx

```diff
@@ -5,19 +5,17 @@ import { PluginPropsContext } from '../../utils/utils.plugin';
 import { PageOne } from '../../pages/PageOne';
 import { PageTwo } from '../../pages/PageTwo';
 import { PageThree } from '../../pages/PageThree';
-import { PageFour } from '../../pages/PageFour';
 import { ROUTES } from '../../constants';
 
 export class App extends React.PureComponent<AppRootProps> {
   render() {
     return (
       <PluginPropsContext.Provider value={this.props}>
         <Routes>
-          <Route path={ROUTES.Two} element={<PageTwo />} />
-          <Route path={`${ROUTES.Three}/:id?`} element={<PageThree />} />
+          <Route path={`${ROUTES.Two}/:id?`} element={<PageTwo />} />
 
           {/* Full-width page (this page will have no navigation bar) */}
-          <Route path={ROUTES.Four} element={<PageFour />} />
+          <Route path={ROUTES.Three} element={<PageThree />} />
```

Comment:
What happened to page 4?

Reply:
I just killed it :)

Reply:
Didn't feel like it brought any value since we have changed how the navigation works.